<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Call</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/config/system_call.png</normaloff>:/config/system_call.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_skin">
      <attribute name="title">
       <string>面板</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_skin">
       <item>
        <layout class="QHBoxLayout" name="skin_layout" stretch="1,2">
         <item>
          <widget class="QListWidget" name="toolList_widget"/>
         </item>
         <item>
          <widget class="QWidget" name="settingsContainer_widget" native="true"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Layout">
      <attribute name="title">
       <string>主題</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_theme">
       <item>
        <widget class="QGroupBox" name="saveTheme_groupBox">
         <property name="title">
          <string>儲存新佈局</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_save">
          <item>
           <widget class="QLabel" name="themeHint">
            <property name="text">
             <string>將目前桌面上所有元件的擺放位置與狀態儲存為新主題。</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vSpacer1">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_theme">
            <item row="0" column="0">
             <widget class="QLabel" name="label_tn">
              <property name="text">
               <string>主題名稱:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="themeName_lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="saveTheme_button">
            <property name="text">
             <string>儲存目前佈局</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="manageTheme_groupBox">
         <property name="title">
          <string>已儲存的主題</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_manage">
          <item>
           <widget class="QListWidget" name="themeList_widget"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_themeBtns">
            <item>
             <widget class="QPushButton" name="loadTheme_button">
              <property name="text">
               <string>載入</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteTheme_button">
              <property name="text">
               <string>刪除</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="vSpacer2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_setting">
      <attribute name="title">
       <string>設定</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_setting">
       <item>
        <widget class="QGroupBox" name="global_groupBox">
         <property name="title">
          <string>全域行為</string>
         </property>
         <layout class="QFormLayout" name="formLayout_global">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="autoStart_checkBox">
            <property name="text">
             <string>隨電腦開機自動啟動</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="trayIcon_checkBox">
            <property name="text">
             <string>在通知區域(System Tray)顯示圖示</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="globalLockDrag_checkBox">
            <property name="text">
             <string>全域禁止拖曳 (覆蓋所有元件設定)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_globalInterval">
            <property name="text">
             <string>全域更新頻率:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_interval">
            <item>
             <widget class="QComboBox" name="globalInterval_comboBox">
              <item>
               <property name="text">
                <string>慢速 (2000ms)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>普通 (1000ms)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>快速 (500ms)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>極速 (250ms)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>即時 (100ms)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>自訂...</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="globalInterval_spinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>50</number>
              </property>
              <property name="maximum">
               <number>60000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vSpacer3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_settingBtns">
         <item>
          <widget class="QPushButton" name="resetAll_button">
           <property name="text">
            <string>重設所有設定</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpacer1">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="applySetting_button">
           <property name="text">
            <string>套用設定</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
